make/stratos/_env

```diff
@@ -0,0 +1,11 @@
+#!/usr/bin/env bash
+
+set -o errexit -o nounset
+
+git_root=$(git rev-parse --show-toplevel)
```

Comment:
I believe elsewhere we let this be overridable via the environment (since we occasionally run various scripts in things that may not have `git` available).